File: app/src/main/java/com/neome/java/api/home/base/dto/DtoMessageReplyPayloadAudio.java
// neome.ai API - do not change
//

package com.neome.java.api.home.base.dto;

import com.neome.java.api.meta.base.Types.MediaIdAudio;

@SuppressWarnings("unused")
public class DtoMessageReplyPayloadAudio extends DtoMessageReplyPayload
{
  public long durationMs;

  public MediaIdAudio mediaIdAudio;
}

File: app/src/main/java/com/neome/api/home/base/dto/DtoMessageReplyPayloadAudio.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.home.base.dto

import com.neome.api.meta.base.Types.MediaIdAudio
import kotlin.properties.Delegates

open class DtoMessageReplyPayloadAudio : DtoMessageReplyPayload() {
    var durationMs: Number by Delegates.notNull<Number>()
    lateinit var mediaIdAudio: MediaIdAudio
}
